tests/unit/test_interface.py

```diff
@@ -94,3 +101,50 @@ def test_validate_source_compatibility_wrong_directory(tmp_path: Path) -> None:
     bad_path.mkdir()
     with pytest.raises(FileNotFoundError, match="Could not find 'decennial_census' in"):
         validate_source_compatibility(bad_path, CENSUS)
+
+
+def test_set_up_dask_client_default() -> None:
+
+    # There should be no dask client yet
+    with pytest.raises(ValueError):
+        client = get_client()
+
+    set_up_dask_client()
+    client = get_client()
+    workers = client.scheduler_info()["workers"]  # type: ignore[no-untyped-call]
+    assert len(workers) == CPU_COUNT
+    assert all(worker["nthreads"] == 1 for worker in workers.values())
+    if is_on_slurm():
+        try:
+            available_memory = float(os.environ["SLURM_MEM_PER_NODE"]) / 1024
+        except KeyError:
+            raise RuntimeError(
+                "You are on Slurm but SLURM_MEM_PER_NODE is not set. "
```

Comment:
Happy to work on this message. It only shows up if you run `pytest` while SSHed into a cluster node (the obvious example being you're using vscode on the cluster)